File: ShoeStore/ShoeStore.Domain/Entities/Comment.cs
namespace ShoeStore.Domain.Entities;

public class Comment
{
    public long Id { get; set; }
    public long UserId { get; set; }
    public int ProductId { get; set; }
    public string Content { get; set; } = null!;
    public byte? Rating { get; set; }
    public DateTime CreatedAt { get; set; }

    public User User { get; set; } = null!;
    public Product Product { get; set; } = null!;
}

File: shoestorefe/lib/domain/entities/comment.dart
class Comment {
  final int id;
  final int userId;
  final String userName;
  final int productId;
  final String productName;
  final String content;
  final int? rating;
  final DateTime createdAt;

  Comment({
    required this.id,
    required this.userId,
    required this.userName,
    required this.productId,
    required this.productName,
    required this.content,
    this.rating,
    required this.createdAt,
  });
}


